Adds documentation about privacy policies. (#248)
medic/cht-docs: content/en/apps/guides/security/privacy-policy.md

---
title: "App Privacy Policies"
linkTitle: "Privacy Policies"
weight: 100
description: >
  Add privacy policies that users must accept before being allowed to use the app
---

As of `3.10.0`, privacy policies can be customized for every language, by adding desired content into HTML files. 
Add these HTML files to the `privacy-policies` folder in your configuration, and associate them to the correct language in the `privacy-policies.json` file.
```json
{
  "en": "en.attachment.html",
  "fr": "fr.file.html",

>   "sw": "swahili.html",
>    ...

Reviewer: Consider removing the `...`. I'm not sure how it shows in the docs site, but in GHs markdown it shows as invalid JSON.
```suggestion
  "sw": "swahili.html"
```

}
```

{{< alert title="Note" >}}
- Use language codes when associating HTML files to languages  
- Files that are not in HTML format will be skipped  
- Files that are not associated with a language will not be uploaded  
{{< /alert >}}

### Adding and Editing a Privacy Policy

There are two ways to add or edit a privacy policy:
 
1. Build the privacy policies into the application with the `upload-privacy-policies` action in `medic-conf`.
```bash

> medic-conf --local upload-resources

Reviewer: ```suggestion
medic-conf --local upload-privacy-policies
```

Author: I can't believe I missed this, thanks

```

2. Update and view privacy policies in the [Admin Console]({{< relref "apps/features/admin" >}}), under `Display` > `Privacy Policies`

### View In Webapp

When a privacy policy is configured for a language, users who load the app in this language are prompted to accept the policy. 

![Accept privacy policy](accept-mobile.png)

> The app will load normally after acceptance. To opt out, a user has to close their app or browser. Once accepted, users will not be prompted to accept the same policy again. However, users will be prompted for acceptance again when the policy is updated or when they change their language. An acceptance log is saved in the `user-settings` file and synced to the server, containing a history of privacy policies accepted by the user.    

Reviewer: Maybe make it clear the app cannot be used until the user has accepted the privacy policy, eg:

```suggestion
The app will load normally after acceptance. The user cannot opt out or skip acceptance meaning if a privacy policy is configured the app will be unusable until the user has accepted it. Once accepted, users will not be prompted to accept the same policy again. However, users will be prompted for acceptance again when the policy is updated or when they change their language. An acceptance log is saved in the `user-settings` file and synced to the server, containing a history of privacy policies accepted by the user.    
```

---
title: "App Privacy Policies"
linkTitle: "Privacy Policies"
weight: 100
description: >
  Add privacy policies that users must accept before being allowed to use the app
---

> As of `3.10.0`, privacy policies can be customized for every language, by adding desired contents into HTML files. 

Reviewer 2: ```suggestion
As of `3.10.0`, privacy policies can be customized for every language, by adding desired content into HTML files. 
```

Add these HTML files to the `privacy-policies` folder in your configuration, and associate them to the correct language in the `privacy-policies.json` file.
```json
{
  "en": "en.attachment.html",
  "fr": "fr.file.html",
  "sw": "swahili.html",
   ...
}
```

{{< alert title="Note" >}}

> Use language codes when associating html files to languages  
> Files that are not in HTML format will be skipped  
> Files that are not associated with a language will not be uploaded  

Reviewer 2: ```suggestion
- Use language codes when associating HTML files to languages  
- Files that are not in HTML format will be skipped  
- Files that are not associated with a language will not be uploaded  
```

{{< /alert >}}

### Build  
Build the privacy policies into the application with the `upload-privacy-policies` action in `medic-conf`.

`medic-conf --local upload-resources`

### Update in the admin console

> Privacy policies can also be viewed and edited in the Admin Console, under `Display` > `Privacy Policies`

Reviewer 2: ```suggestion
Privacy policies can also be viewed and edited in the [Admin Console]({{ < relref "apps/features/admin" >}}), under `Display` > `Privacy Policies`
```

> Privacy policies can also be viewed and edited in the Admin Console, under `Display` > `Privacy Policies`

Reviewer 2: Would be good to consolidate under one heading, such as:

### Adding and Editing a Privacy Policy
There are two ways to add and edit a privacy policy:
1) Build the privacy policies....
2) Update and view the privacy policy in the Admin Console...

> ### In webapp

Reviewer 2: ```suggestion
### View In Webapp
```

When a privacy policy is configured for a language, users who load the app in this language are prompted to accept the policy. 

> ![Accept privacy policy](accept-mobile.jpg)

Reviewer 2: Image is not rendering for me. Try following: https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/#images

Author: Ah oops, I think It forgot to commit it, sorry

The app will load normally after acceptance. 
To opt out, a user has to close their app or browser.  
Once accepted, users will not be prompted to accept the same policy again. However, users will be prompted for acceptance again when the policy is updated or when they change their language.  

> An acceptance log is saved in the user-settings file and synced to the server, containing a history of privacy policies accepted by the user.    

Reviewer 2: I think this can all be combined into a single paragraph (instead of a separate line for each sentence)
